ds-collector/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,221 @@
+# Troubleshooting the Collector
+The Collector has been designed and built to handle a wide variety of software versions and environmental topologies; this is part of why the configuration has so many options. The purpose of this document is to help provide some techniques to identifying problems encountered when running the collector, and include some work-arounds.
+
+This document assumes you have some basic training and understanding of configuration and Collector execution - you know how to do a "happy path" collection on the toplogy (ssh/Docker/Kubernetes) being used.
+
+---
+ℹ️ It is strongly advised that you use the `-v` option when invoking `ds-collector`, as this will capture the output into a dated log file in the base directory (e.g. `/tmp/datastax/ds-collector-2022-05-26-08-46-1653554777.log`). This file is more easily used by you to investigate the problems, and to send to others who may be able to help you.
+
+---
+
+## Review of Process
+As a reminder, the high-level collector process after invoking `ds-collector` is:
+
+1. Do basic checks on the Bastion: can we make `baseDir` (e.g. `/tmp/datastax`), and do we have the right commands available?
+2. If doing `-T` test mode:
+    - Connect to the `-n` Node if it is not a file, and get list of nodes via `nodetool status`
+    - Confirm the command `date` can be invoked on each Node
+3. Connect to the `-n` Node if it is not a file, and get list of nodes via `nodetool status`
+4. For each Node in the list:
+    - Transfer a number of files from the Bastion to the Node
+    - Run the collection script on the node
+    - Pull the collected `.tar.gz` tarball back to the Bastion
+    - Remove collected `.tar.gz` from the Node
+5. For each collected `.tar.gz` file on the Bastion:
+    - Encrpyt the file if so configured
+    - Send the file to S3 if so configured
+    - Remove the file if it has been transferred to S3, and if `keepArtifact=` is not configured, or set to `false`.
+    - Files not uploaded will remain on the Bastion in the `baseDir`
```

Comment:
(trivial) will this section be difficult over time to keep accurate? 

(trivial) one of your diagrams here would be very valuable! i can even see this section becoming it's own `PROCESS.md` doc file, with all your images included…